py/setup.py

```diff
@@ -239,7 +239,7 @@ def run(self):
         libraries=["torchtrt"],
         include_dirs=[
             dir_path + "torch_tensorrt/csrc", dir_path + "torch_tensorrt/include",
-            dir_path + "/../bazel-TRTorch/external/tensorrt/include",
+            dir_path + "/../bazel-Torch-TensorRT/external/tensorrt/include",
```

Comment:
This line is already included on the next line

Reply:
rectified it